index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <!-- <meta http-equiv="X-UA-Compatible" content="IE=edge" /> -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <nav>
+      <div class="container">
+        <div class="name-info">Sunday Stanley</div>
+        <div class="menu-toggle">
+          <img src="./image/menu.png" alt="menu bar" />
+        </div>
+        <ul class="list-container">
+          <li>
+            <a href="">Portfolio</a>
+          </li>
+          <li>
+            <a href=""> About </a>
+          </li>
+          <li>
+            <a href="">Contact</a>
+          </li>
+          <li>
+            <a href=""><img src="./image/msg.png" alt="msg icon" /></a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+    <main>
+      <div class="bio-container">
+        <h1>Hey There. I’m Stanley</h1>
+        <h3>I’am a Software Developer</h3>
+        <div class="paragragh">
+          <p
+            >I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.</p
+          >
+        </div>
+        <div class="icon">
+          <img src="./image/GitIcon.png" alt="git icon" />
+          <img src="./image/LinkedInIcon.png" alt="linkedin icon" />
+          <img src="./image/finIcon.png" alt="fin icon" />
+          <img src="./image/twitterIcon.png" alt="twitter icon" />
+          <img src="./image/mIcon.png" alt="m icon" />
+        </div>
```

Comment:
- [ ] Kindly use `ul` instead of `div` and wrap each image with `li` and `a`, to link it to your accounts.

like 
```
<ul>
  <li><a href=""><img ..../><a/></li>
  <li><a href=""><img ..../><a/></li>
  <li><a href=""><img ..../><a/></li>
  ............
</ul>
```

Reply:
modified the tags